src/couch_httpd_auth.erl

```diff
@@ -304,12 +310,19 @@ handle_session_req(#httpd{method='POST', mochi_req=MochiReq}=Req, AuthModule) ->
             UserSalt = couch_util:get_value(<<"salt">>, UserProps2),
             CurrentTime = make_cookie_time(),
             Cookie = cookie_auth_cookie(Req, ?b2l(UserName), <<Secret/binary, UserSalt/binary>>, CurrentTime),
+            % maybe set the csrf token
+            Cookies = case couch_httpd:qs_value(Req, "csrf", "false") of
```

Comment:
I meant https://en.wikipedia.org/wiki/Cross-site_request_forgery#Cookie-to-Header_Token earlier, sorry for confusion.